torchvision/datasets/voc.py

```diff
@@ -83,7 +83,10 @@ def __init__(self,
         self.url = DATASET_YEAR_DICT[year]['url']
         self.filename = DATASET_YEAR_DICT[year]['filename']
         self.md5 = DATASET_YEAR_DICT[year]['md5']
-        self.image_set = image_set
+        # FIXME: Is it common to not have the *.txt of the image_set in the root folder?
+        #  Previous code checked for this, but only reported a wrong value for image_set
```

Comment:
Not common, and not even supported in the current implementation, so we are good here.